### PR TITLE
Controller spec

### PR DESCRIPTION
**Pull request by Said-laasri**
**Added  RSpec as a gem to the project.**
**Created a Request spec file for all controllers.**
**for each action checking:**

- > If the response status was correct.
- > If a correct template was rendered.
- > If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/Request/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    end
+  end
+end
+
+# Path: spec/Request/user_spec.rb
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET show' do
+    before(:example) do
+      get '/users/7'
```
Comment (GzimAsani) [inline spec/Request/user_spec.rb]: - [x] 9 of your tests are failing, kindly update your tests. Hint: you shouldn't hardcore your users or posts on your tests.
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello    @Said-laasri             :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with styling, it matches the design :muscle: 

 ## Required Changes 

### Note this project should be done as one project react/rails.
- [x]  Please update your readme file and add the comment that the users can run in order to test your app
- [x] Check the comments below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/Request/user_spec.rb (hunk 1 above)
Comment (Said-laasri) [reply spec/Request/user_spec.rb]: @GzimAsani  Thank you for reviewing my project, I will make the necessary corrections as suggested

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Said-laasri , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Said-laasri** [comment]
@VICTOR-CHUKWU Thank you for reviewing my project,

**Pull request merged**